src/_P076_HLW8012.ino

```diff
@@ -165,25 +240,56 @@ boolean Plugin_076(byte function, struct EventStruct *event, String &string) {
     SaveCustomTaskSettings(event->TaskIndex, (byte *)&hlwMultipliers,
                            sizeof(hlwMultipliers));
 
-   Settings.TaskDevicePluginConfig[event->TaskIndex][4] = getFormItemInt(F("p076_current_read"));
-   Settings.TaskDevicePluginConfig[event->TaskIndex][5] = getFormItemInt(F("p076_cf_trigger"));
-   Settings.TaskDevicePluginConfig[event->TaskIndex][6] = getFormItemInt(F("p076_cf1_trigger"));
-
     if (PLUGIN_076_DEBUG) {
-      String log = F("HLW8012: Saved Calibration from Config Page");
-      addLog(LOG_LEVEL_INFO, log);
+     String log = F("HLW8012: Saved Calibration from Config Page");
+     addLog(LOG_LEVEL_INFO, log);
     }
 
     if (Plugin_076_hlw) {
-      Plugin_076_hlw->setCurrentMultiplier(hlwMultipliers[0]);
-      Plugin_076_hlw->setVoltageMultiplier(hlwMultipliers[1]);
-      Plugin_076_hlw->setPowerMultiplier(hlwMultipliers[2]);
+     Plugin_076_hlw->setCurrentMultiplier(hlwMultipliers[0]);
+     Plugin_076_hlw->setVoltageMultiplier(hlwMultipliers[1]);
+     Plugin_076_hlw->setPowerMultiplier(hlwMultipliers[2]);
     }
 
     if (PLUGIN_076_DEBUG) {
-      String log = F("HLW8012: Multipliers Reassigned");
+     String log = F("HLW8012: Multipliers Reassigned");
+     addLog(LOG_LEVEL_INFO, log);
+    }
+
+    //Set Pin settings
+    byte selectedDevice = getFormItemInt(F("p076_predefined_settings"));
```

Comment:
Try to shorten the internal identifiers.
The compiled binary is getting too big, so we must try to de-duplicate code as much as possible and since those internal identifiers are just used to recognize returned values from the web interface, they can be made shorter. No human is reading them, only when debugging.

Reply:
Fixed, shorten to p076_preDefDevSel